packages/patternfly-4/react-table/src/components/Table/Table.d.ts

```diff
@@ -58,6 +58,8 @@ export interface TableProps extends Omit<Omit<HTMLProps<HTMLTableElement>, 'onSe
   onSelect?: Function;
   onSort?: Function;
   actions?: Array<IAction | ISeparator>;
+  actionResolver?: (rowData: Object, extraParams: Object) => Array<IAction | ISeparator>;
```

Comment:
ok - yes, re-reading this piece, it does resolve my issue. `rowData` is returned in the resolver thus letting the consumer construct custom JSX w/ `rowData` for each action.